### ISSUE TITLE
Glimmer Component Support

### ISSUE THREAD

**Issue by runspired** [ember-best-practices/ember-perf-timeline#49]
As the title says :D

**Comment by SergeAstapov** [comment]
+1

**Comment by SergeAstapov** [comment]
Maybe related https://github.com/emberjs/rfcs/pull/778

**Comment by michaelbdai** [comment]
ember-perf-timeline is not fully supported by glimmer component.
The ember-perf-timeline uses the subscribe API from ember/instrumentation.
https://github.com/ember-best-practices/ember-perf-timeline/blob/master/addon/initializers/ember-perf-timeline.js#L146
It tries to get `payload.object` and `payload.initialRender`.
https://github.com/ember-best-practices/ember-perf-timeline/blob/master/addon/initializers/ember-perf-timeline.js#L8
glimmer does send payload with object key. but does not have `initialRender`. Then `initialRender` is always false, and it always display it as "update". However, it should still correctly display text `EmberRender:${payload.object}`
I will try to fix this issue when implementing https://github.com/emberjs/rfcs/pull/778

**Comment by michaelbdai** [comment]
> Maybe related [emberjs/rfcs#778](https://github.com/emberjs/rfcs/pull/778)

Yes, they are related. I can help look into this issue when implementing the https://github.com/emberjs/rfcs/pull/778

**Comment by SergeAstapov** [comment]
That would be awesome @michaelbdai !
